### ISSUE TITLE
Plugin with id 'kotlin-android-extensions' not found.

### ISSUE THREAD

**Issue by ghost** [redalpha01/cordova-plugin-measure#1]
Hi Im getting Plugin with id 'kotlin-android-extensions' not found. when building the apk 

android 9.

**Comment by redalpha01** [comment]
I'm not the original creator of this plugin but I can try to help you. Did you install this plugin through cordova?

**Comment by ghost** [comment]
Yes , installed it via cordova. Can't find any documentation about this :(

**Comment by ghost** [comment]
Hope you can help me

**Comment by redalpha01** [comment]
Try this perhaps https://developer.aliyun.com/mirror/npm/package/cordova-support-kotlin

**Comment by redalpha01** [comment]
Did you try it? Did it help?
